app/routers/cultures.py

```diff
@@ -0,0 +1,203 @@
+
+    - **id**: unique identifier
+    - **plot_number**: plot_number associated with the culture
+    - **production_code**: production_code associated with the culture
+    - **start_date**: each start_date have a date
+    - **end_date**: each end_date have a date
+    - **quantity**: each quantity have integer value
+    """
+    return {"data": fetch_all_culture()}
+
+
+@router.post("/", status_code=status.HTTP_201_CREATED, summary="Create one culture")
+def create_culture(culture: Culture):
+    """
+    Create a culture with all the information:
+
+    - **plot_number**: plot_number associated with the culture
+    - **production_code**: production_code associated with the culture
+    - **start_date**: each start_date have a date
+    - **end_date**: each end_date have a date
+    - **quantity**: each quantity have integer value
+    """
+    if not culture.id:
+        raise HTTPException(
+            status_code=400,
+            detail="Culture is empty",
+            headers={"X-Error": "Resource empty"},
+        )
+    db_culture = fetch_culture_by_id(culture_id=culture.id)
+    if db_culture is not None:
+        raise HTTPException(
+            status_code=409,
+            detail="Culture already exists",
+            headers={"X-Error": "Resource already exists"},
+        )
+    return insert_culture(
+        plot_number=culture.plot_number,
+        production_code=culture.production_code,
+        start_date=culture.start_date,
+        end_date=culture.end_date,
+        quantity=culture.quantity,
+    )
+
+
+@router.get(
+    "/{culture_id}", status_code=status.HTTP_200_OK, summary="Fetch one culture"
+)
+def read_culture(culture_id: UUID):
+    """
+    Fetch one culture with all the information:
+
+    - **id**: unique identifier
+    - **plot_number**: plot_number associated with the culture
+    - **production_code**: production_code associated with the culture
+    - **start_date**: each start_date have a date
+    - **end_date**: each end_date have a date
+    - **quantity**: each quantity have integer value
+
+    :parameter culture_id:
+    """
+    db_culture = fetch_culture_by_id(culture_id=culture_id)
+    if not db_culture:
+        raise HTTPException(
+            status_code=404,
+            detail="Culture not found",
+            headers={"X-Error": "Resource not found"},
+        )
+    return {"data": db_culture}
+
+
+@router.put(
+    "/{culture_id}",
+    status_code=status.HTTP_204_NO_CONTENT,
+    summary="Update a culture",
+)
+def update_culture_by_id(culture_id: UUID, culture: CreateUpdateCulture):
+    """
+    Update one culture using his id value
+
+    :parameter culture_id:
+    :parameter culture:
+    """
+    db_culture = fetch_culture_by_id(culture_id=culture_id)
+    if not db_culture:
+        raise HTTPException(
+            status_code=404,
+            detail="Culture not found",
+            headers={"X-Error": "Resource not found"},
+        )
+    return update_culture(
```

Comment:
Need concrete return for front-end developers.

Reply:
Example ?